models/transfers/celo/erc20/transfers_celo_erc20.sql

```diff
@@ -0,0 +1,108 @@
+{{ 
+    config(
+        tags = ['dunesql'],
+        alias = alias('erc20'),
+        partition_by = ['block_month'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['block_time', 'tx_hash', 'index', 'wallet_address', 'amount_raw'],
+        post_hook='{{ expose_spells(\'["celo"]\',
+                                    "sector",
+                                    "transfers",
+                                    \'["soispoke", "dot2dotseurat", "tschubotz", "tomfutago"]\') }}'
+    )
+}}
+
+with
+    sent_transfers as (
+        select
+            to as wallet_address,
+            contract_address as token_address,
+            evt_block_time as block_time,
+            date_trunc('month', evt_block_time) as block_month,
+            cast(value as double) as amount_raw,
+            evt_index as index,
+            evt_tx_hash as tx_hash
```

Comment:
We can add a `type` here for each union part:
```suggestion
            evt_index,
            evt_tx_hash as tx_hash,
            'send' as type
```
Also we can use `evt_index` naming for consistency.